modular_skyrat/master_files/code/modules/mob/living/carbon/human.dm

```diff
@@ -1,13 +1,19 @@
 /mob/living/carbon/human/ZImpactDamage(turf/T, levels)
-	//Non cat-people smash into the ground
-	if(!isfelinid(src) && !istajaran(src))
+	// Cat-people always land on their feet
+	if(isfelinid(src) || istajaran(src))
+		//Check to make sure legs are working
+		var/obj/item/bodypart/left_leg = get_bodypart(BODY_ZONE_L_LEG)
+		var/obj/item/bodypart/right_leg = get_bodypart(BODY_ZONE_R_LEG)
+		if(!left_leg || !right_leg || left_leg.bodypart_disabled || right_leg.bodypart_disabled)
+			return ..()
+		Stun(0.5 SECONDS) // You can't move and you can't use items
+		//Nailed it!
+		visible_message(span_notice("[src] lands elegantly on [p_their()] feet!"),
+			span_warning("You fall [levels] level[levels > 1 ? "s" : ""] into [T], perfecting the landing!"))
```

Comment:
i always wondered

was there a way to make that 1 one?
because seeing funny number not being a word number annoyed me

Reply:
Yeah, we can probably use something for that, hold on.